Playground/index-local.html

```diff
@@ -22,6 +22,7 @@
     <script src="../dist/preview%20release/Oimo.js"></script>
     <script src="../dist/preview%20release/gltf_validator.js"></script>
     <script src="../dist/preview%20release/earcut.min.js"></script>
+    <script src="../dist/preview%20release/basisTranscoder/basis_transcoder.js"></script>

```

Comment:
we need to do it for all html files (full, frame, index, etc)

Reply:
I removed it from here to load the script using js like draco seems to work